Error installing latest release 2021.8

Hi,

I'm now getting this error when installing numpy-quaternion

```
pip install numpy-quaternion
```

I'm using a Ubuntu 18.04 machine

```Collecting numpy-quaternion
Using cached numpy-quaternion-2021.8.25.12.59.41.tar.gz (58 kB)
Installing build dependencies ... done
Getting requirements to build wheel ... done
Preparing wheel metadata ... done
Collecting numpy>=1.13
Using cached numpy-1.16.6-cp27-cp27mu-manylinux1_x86_64.whl (17.0 MB)
Building wheels for collected packages: numpy-quaternion
Building wheel for numpy-quaternion (PEP 517) ... error
ERROR: Command errored out with exit status 1:
command: /home/techlabs/Workspace/simulation_of_complex_interactions/simulation_ws/bundle/cache/bundle_staging/usr/bin/python2 /home/techlabs/.local/lib/python2.7/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /tmp/tmpPpqsRx
cwd: /tmp/pip-install-MJWC2w/numpy-quaternion
Complete output (21 lines):
running bdist_wheel
running build
running build_py
creating build
creating build/lib.linux-x86_64-2.7
creating build/lib.linux-x86_64-2.7/quaternion
copying src/quaternion/means.py -> build/lib.linux-x86_64-2.7/quaternion
copying src/quaternion/quaternion_time_series.py -> build/lib.linux-x86_64-2.7/quaternion
copying src/quaternion/numba_wrapper.py -> build/lib.linux-x86_64-2.7/quaternion
copying src/quaternion/init.py -> build/lib.linux-x86_64-2.7/quaternion
copying src/quaternion/calculus.py -> build/lib.linux-x86_64-2.7/quaternion
running build_ext
building 'quaternion.numpy_quaternion' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/src
x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-gnDdqE/python2.7-2.7.17=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/tmp/pip-build-env-6v5nfb/overlay/lib/python2.7/site-packages/numpy/core/include -Isrc -I/usr/include/python2.7 -c src/quaternion.c -o build/temp.linux-x86_64-2.7/src/quaternion.o -O3 -w
x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-gnDdqE/python2.7-2.7.17=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/tmp/pip-build-env-6v5nfb/overlay/lib/python2.7/site-packages/numpy/core/include -Isrc -I/usr/include/python2.7 -c src/numpy_quaternion.c -o build/temp.linux-x86_64-2.7/src/numpy_quaternion.o -O3 -w
src/numpy_quaternion.c:786:19: error: unknown type name ‘Py_hash_t’
static NPY_INLINE Py_hash_t
^~~~~~~~~
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
ERROR: Failed building wheel for numpy-quaternion
Failed to build numpy-quaternion
ERROR: Could not build wheels for numpy-quaternion which use PEP 517 and cannot be installed directly
[186.244s] ERROR:colcon:colcon bundle: Command '['/home/techlabs/Workspace/simulation_of_complex_interactions/simulation_ws/bundle/cache/bundle_staging/usr/bin/python2', '-m', 'pip', 'install', '--default-timeout=100', '--ignore-installed', '-r', '/home/techlabs/Workspace/simulation_of_complex_interactions/simulation_ws/bundle/cache/installer_cache/pip/requirements']' returned non-zero exit status 1.
Traceback (most recent call last):
File "/usr/local/lib/python3.6/dist-packages/colcon_core/command.py", line 528, in verb_main
rc = context.args.main(context=context)
File "/usr/local/lib/python3.6/dist-packages/colcon_bundle/verb/bundle.py", line 138, in main
upgrade_deps_graph)
File "/usr/local/lib/python3.6/dist-packages/colcon_bundle/verb/bundle.py", line 187, in _manage_dependencies
include_sources=context.args.include_sources)
File "/usr/local/lib/python3.6/dist-packages/colcon_bundle/verb/_installer_manager.py", line 78, in run_installers
installer_metadata[name] = installer.install()
File "/usr/local/lib/python3.6/dist-packages/colcon_bundle/installer/base_pip_installer.py", line 100, in install
subprocess.check_call(pip_args)
File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/home/techlabs/Workspace/simulation_of_complex_interactions/simulation_ws/bundle/cache/bundle_staging/usr/bin/python2', '-m', 'pip', 'install', '--default-timeout=100', '--ignore-installed', '-r', '/home/techlabs/Workspace/simulation_of_complex_interactions/simulation_ws/bundle/cache/installer_cache/pip/requirements']' returned non-zero exit status 1.
```
I am not getting this error if I try to download the previous version 2021.7.28.16.11.54

Is anyone aware of this issue or this local to me? Any help would be appreciated.

Best regards,

--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,7 @@ import numpy as np
 
 
 # Set this first for easier replacement
-version = "2021.8.25.12.59.41"
+version = "2021.8.26.15.40.13"
 
 if "win" in platform.lower() and not "darwin" in platform.lower():
     extra_compile_args = ["/O2"]

--- a/src/numpy_quaternion.c
+++ b/src/numpy_quaternion.c
@@ -783,6 +783,9 @@ pyquaternion_richcompare(PyObject* a, PyObject* b, int op)
 #if PY_VERSION_HEX > 0x030a00a6
 #define _newpy_HashDouble _Py_HashDouble
 #else
+#if PY_VERSION_HEX < 0x030200a1
+#define Py_hash_t long
+#endif
 static NPY_INLINE Py_hash_t
 _newpy_HashDouble(PyObject *NPY_UNUSED(ignored), double val)
 {

--- a/src/quaternion/__init__.py
+++ b/src/quaternion/__init__.py
@@ -1,7 +1,7 @@
 # Copyright (c) 2020, Michael Boyle
 # See LICENSE file for details: <https://github.com/moble/quaternion/blob/main/LICENSE>
 
-__version__ = "2021.8.25.12.59.41"
+__version__ = "2021.8.26.15.40.13"
 __doc_title__ = "Quaternion dtype for NumPy"
 __doc__ = "Adds a quaternion dtype to NumPy."
 __all__ = ['quaternion',

